### PR TITLE
Expose salt length and hash name fields

### PR DESCRIPTION
**Pull request by armfazh**
changes:
 - Expose salt length and hash name fields
 - Updates dependencies
 - format files

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -5,11 +5,8 @@
     "testEnvironment": "node",
     "transform": {},
     "setupFiles": [
-        "./test/jest.setup-file.js"
+        "./lib/test/jest.setup-file.js"
```
Comment (thibmeu) [inline jest.config.json:8]: nits: might want to use `ts-node` or `ts-jest` instead

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@cloudflare/blindrsa-ts",
-    "version": "0.1.2",
+    "version": "0.1.3",
```
Comment (thibmeu) [inline package.json:3]: don't update the version without a release, as you don't know which commits are part of it yet

File: src/blindrsa.ts [hunk 3]
```diff
@@ -26,8 +26,8 @@ export class BlindRSA {
     private static readonly NAME = 'RSA-PSS';
 
     constructor(
-        private readonly hash: string,
-        private readonly saltLength: number,
+        public readonly hash: string,
```
Comment (thibmeu) [inline src/blindrsa.ts:29]: flagging for info: this same line is modified in #7 . It's going to create merge conflict

File: test/blindrsa.test.ts [hunk 4]
```diff
@@ -78,6 +78,21 @@ async function keysFromVector(v: Vector, extractable: boolean): Promise<CryptoKe
     return { privateKey, publicKey };
 }
 
+test('Parameters', () => {
+    const hash = 'SHA-384';
+    const vectorList = [
+        { hash, saltLength: 0x30, suite: SUITES.SHA384.PSS.Deterministic() },
```
Comment (thibmeu) [inline test/blindrsa.test.ts:84]: probably need to merge before/after #7 and rename `SUITES` accordingly

File: test/blindrsa.test.ts [hunk 5]
```diff
@@ -78,6 +78,21 @@ async function keysFromVector(v: Vector, extractable: boolean): Promise<CryptoKe
     return { privateKey, publicKey };
 }
 
+test('Parameters', () => {
+    const hash = 'SHA-384';
+    const vectorList = [
```
Comment (thibmeu) [inline test/blindrsa.test.ts:83]: document where these vectors come from

**Review by thibmeu — COMMENTED** [review]
the PR on it's own is good. However, I feel it's going to be redone with #7 (specifically this line https://github.com/cloudflare/blindrsa-ts/pull/7/files#diff-2638c61a72669eb3fc1928e638c58f24203cc64d24ca6d0a1744ecd69d9aece4R35)
Maybe you need to declare `hash` and `saltLength` as getter methods instead of public readonly parameters. This would provide a clear path towards resolving the two approaches.

File: package.json (hunk 2 above)
Comment (armfazh) [reply package.json:3]: correct, fixed now

File: test/blindrsa.test.ts (hunk 5 above)
Comment (armfazh) [reply test/blindrsa.test.ts:83]: done

File: jest.config.json (hunk 1 above)
Comment (armfazh) [reply jest.config.json:8]: ts-node sometimes hide errors triggered by the `tsc` compiler, and one other dependency to track

File: test/blindrsa.test.ts (hunk 4 above)
Comment (armfazh) [reply test/blindrsa.test.ts:85]: that will be solved in the other PR.

**Comment by armfazh** [comment]
> Maybe you need to declare hash and saltLength as getter methods instead of public readonly parameters. This would provide a clear path towards resolving the two approaches.

I will consider add getters in the other PR. 
let's finish this one. ready for a second review.

**Review by thibmeu — APPROVED** [review]

**Pull request merged**